src/main/java/de/unistuttgart/overworldbackend/data/Position.java

```diff
@@ -0,0 +1,23 @@
+package de.unistuttgart.overworldbackend.data;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Position {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    UUID id;
+
+    float x;
+    float y;
```

Comment:
I know, it's a bit late now for that, but wouldn't an `int` make more sense in our case (-> tile)?